may. 16, 2023 7:10:07 P. M. pe.edu.upeu.app.util.ErrorLogger log
SEVERE: pe.edu.upeu.app.dao.FacultadDao.update
org.sqlite.SQLiteException: [SQLITE_ERROR] SQL error or missing database (no such table: id_facultad)
	at org.sqlite.core.DB.newSQLException(DB.java:1141)
	at org.sqlite.core.DB.newSQLException(DB.java:1152)
	at org.sqlite.core.DB.throwex(DB.java:1112)
	at org.sqlite.core.NativeDB.prepare_utf8(Native Method)
	at org.sqlite.core.NativeDB.prepare(NativeDB.java:117)
	at org.sqlite.core.DB.prepare(DB.java:267)
	at org.sqlite.core.CorePreparedStatement.<init>(CorePreparedStatement.java:44)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.<init>(JDBC3PreparedStatement.java:30)
	at org.sqlite.jdbc4.JDBC4PreparedStatement.<init>(JDBC4PreparedStatement.java:25)
	at org.sqlite.jdbc4.JDBC4Connection.prepareStatement(JDBC4Connection.java:34)
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:155)
	at org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:135)
	at pe.edu.upeu.app.dao.FacultadDao.update(FacultadDao.java:69)
	at pe.edu.upeu.app.dao.FacultadDao.main(FacultadDao.java:145)

may. 16, 2023 7:11:22 P. M. pe.edu.upeu.app.util.ErrorLogger log
SEVERE: pe.edu.upeu.app.dao.FacultadDao.delete
org.sqlite.SQLiteException: [SQLITE_CONSTRAINT_FOREIGNKEY] A foreign key constraint failed (FOREIGN KEY constraint failed)
	at org.sqlite.core.DB.newSQLException(DB.java:1141)
	at org.sqlite.core.DB.newSQLException(DB.java:1152)
	at org.sqlite.core.DB.execute(DB.java:947)
	at org.sqlite.core.DB.executeUpdate(DB.java:1016)
	at org.sqlite.jdbc3.JDBC3PreparedStatement.executeUpdate(JDBC3PreparedStatement.java:90)
	at pe.edu.upeu.app.dao.FacultadDao.delete(FacultadDao.java:109)
	at pe.edu.upeu.app.dao.FacultadDao.main(FacultadDao.java:151)

